File: Pparty/API/pparty/src/main/java/br/cefet/pparty/model/Usuario.java
package br.cefet.pparty.model;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString


public class Usuario {
    private int idUsuario;
    private Date datanasc;
    private String nome;
    private String email;
    private String senha;
    private String endereco;
    
}

File: Pparty/Implemento-Last/src/app/model/usuario.ts
export class Usuario 
{

    idUsuario: number;
    email: string;
    nome: string;
    senha: string;
    datanasc: String;
    endereco: String;

    constructor()
    {
        this.idUsuario = 0;
        this.email = "";
        this.nome = "";
        this.senha = "";
        this.datanasc = "";
        this.endereco = "";
    }
}
